apps/trading/pages/_app.page.tsx

```diff
@@ -99,6 +103,7 @@ function AppBody({ Component }: AppProps) {
           />
           <ViewingBanner />
           <UpgradeBanner showVersionChange={true} />
+          <MarketSuccessorBanner />
```

Comment:
We should move this banner so it is displayed within the market page, then we dont need the path matching logic